fluentbit/dockerdir/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+set -o xtrace
+
+export PATH=$PATH:/opt/td-agent-bit/bin
+
+if [ -f '/etc/fluentbit/custom/fluentbit_custom.conf' ]; then
+    cp /etc/fluentbit/custom/fluentbit_custom.conf /etc/fluentbit/
```

Comment:
what if we passed several files in the secret?

Reply:
we will use one configmap and secret for env vars

Reply:
one secret can contain many files